pulsar/activity/descriptor.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "pulsar-activity",
+	"type": "flogo:activity",
+	"version": "0.0.1",
+	"title": "Apache Pulsar activity",
+	"description": "A Simple activity to produce messages on apache pulsar",
+	"settings": [
+		{
+			"name": "connection",
+			"type": "any",
```

Comment:
Shouldn't the type be `connection` @fm-tibco ?

Reply:
Yes